src/Database/Adapter/PdoAdapter.php

```diff
@@ -403,6 +408,50 @@ private function isMulti(array $data): bool
         return true;
     }
 
+    private function splitColumnNameAndOperator(string $key, mixed $value, ?string &$columnName, ?string &$operator): void
```

Comment:
mixed typehint is alloed only from php 8. This library supports also PHP >= 7.1 so we need to use just php doc annotation for it like in other cases above

Reply:
Ah! That's why I saw it missing... ahah ok I'll fix that

Reply:
yes it can be :)